src/graphql/mutations/createEtchPacket.js

```diff
@@ -0,0 +1,44 @@
+
+const defaultResponseQuery = `{
```

Comment:
What do we want this to actually be? I just made this up here for me to see interesting stuff about the mutation PR.

Reply:
I think `id, eid, name, documentGroup { id, eid, files, signers { id, eid, name, email } }`. Ignore the template and things for now

Reply:
done!